e2e/content-services/document-list/document-list-selection.e2e.ts

```diff
@@ -0,0 +1,150 @@
+    const folderModel = new FolderModel({ name: 'folder' });
+    const docxFileModel = new FileModel({
+        'name': browser.params.resources.Files.ADF_DOCUMENTS.DOCX.file_name,
+        'location': browser.params.resources.Files.ADF_DOCUMENTS.DOCX.file_location
+    });
+    const displayColumnName = 'Display name';
+
+    this.alfrescoJsApi = new AlfrescoApi({
+        provider: 'ECM',
+        hostEcm: browser.params.testConfig.adf_acs.host
+    });
+
+    beforeAll(async () => {
+        try {
+            await this.alfrescoJsApi.login(browser.params.testConfig.adf.adminEmail, browser.params.testConfig.adf.adminPassword);
+            await this.alfrescoJsApi.core.peopleApi.addPerson(acsUser);
+            await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+
+            await loginPage.loginToContentServicesUsingUserModel(acsUser);
+
+            await contentServicesPage.goToDocumentList();
+            await contentServicesPage.checkDocumentListElementsAreDisplayed();
+            await contentServicesPage.createNewFolder(folderModel.name);
+            await contentServicesPage.uploadFile(docxFileModel.location);
+            await contentServicesPage.checkContentIsDisplayed(docxFileModel.name);
+            await uploadDialog.clickOnCloseButton();
+            await uploadDialog.dialogIsNotDisplayed();
+        } catch (error) {
+            throw new Error(`API call failed in beforeAll: ${error}`);
+        }
+    });
+
+    afterAll(async () => {
+        await navigationBarPage.clickLogoutButton();
+    });
+
+    it('[C274696] Should be able to select and unselect a file or folder', async () => {
+        await contentServicesPage.selectRow(docxFileModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected(displayColumnName, docxFileModel.name);
+
+        await paginationPage.clickItemsPerPageDropdown();
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected(displayColumnName, docxFileModel.name);
+
+        await contentServicesPage.selectRow(docxFileModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsNotSelected(displayColumnName, docxFileModel.name);
+
+        await contentServicesPage.selectRow(folderModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected(displayColumnName, folderModel.name);
+
+        await paginationPage.clickItemsPerPageDropdown();
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected(displayColumnName, folderModel.name);
+
+        await contentServicesPage.selectRow(folderModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsNotSelected(displayColumnName, folderModel.name);
+    });
+
+    it('[C260057] Should be able to choose between the Selection Mode options and select items accordingly', async () => {
+        let list;
```

Comment:
you don't need to declare a variable at the beginning of the method is better closer to the usage. Only vor var kind is a good pratice

Reply:
there are other places dow with the same problem

Reply:
done